### PR TITLE
`Development`: Update Atlassian versions

### PR DESCRIPTION
**Pull request by Mtze**
### Motivation and Context

Atlassian Updates 

- Bitbucket → [8.8.2](https://confluence.atlassian.com/bitbucketserver/bitbucket-data-center-and-server-8-8-release-notes-1182932444.html)  
- Bamboo → [9.2.1](https://confluence.atlassian.com/bambooreleases/bamboo-9-2-release-notes-1207179202.html)  
- Jira → [9.4.3](https://confluence.atlassian.com/jirasoftware/jira-software-9-4-x-release-notes-1178869757.html)

### CODE REVIEW

File: src/main/docker/atlassian.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "2"
 services:
     jira:
         container_name: artemis_jira
-        image: ghcr.io/ls1intum/artemis-jira:8.20.11
+        image: ghcr.io/ls1intum/artemis-jira:9.4.0
```
Comment (Hialus) [inline src/main/docker/atlassian.yml:5]:
```suggestion
        image: ghcr.io/ls1intum/artemis-jira:9.4.3
```

**Review by Hialus — CHANGES_REQUESTED** [review]

**Review by Hialus — DISMISSED** [review]

File: src/main/docker/atlassian.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ services:
                 ipv4_address: 172.20.0.2
     bitbucket:
         container_name: artemis_bitbucket
-        image: ghcr.io/ls1intum/artemis-bitbucket:7.21.4
+        image: ghcr.io/ls1intum/artemis-bitbucket:7.21.10
```
Comment (TheZoker) [inline src/main/docker/atlassian.yml:16]:
```suggestion
        image: ghcr.io/ls1intum/artemis-bitbucket:8.8.2
```

**Review by TheZoker — CHANGES_REQUESTED** [review]
Prelive bitbucket has been updated to `8.8.2`

**Review by TheZoker — APPROVED** [review]
e2e tests run again, so issues seem to be fixed now 🎉

**Pull request merged**